### ISSUE TITLE
Can't retrieve the weight files from the provided google Drive

### ISSUE THREAD

**Issue by yadhu98** [KupynOrest/DeblurGAN#184]
Can't find weight files in the provided google drive. So that we are unable to proceed to the further steps.

**Comment by KenLee12323** [comment]
+1

**Comment by KenLee12323** [comment]
I just found that it is in the checkpoint folder

**Comment by shreya2000** [comment]
> I just found that it is in the checkpoint folder

Hey , did you get the weight file ? I don't think it's inside checkpoint folder. If you have got it ,can you please help me out with the weight file ?
Thanks in advance.

**Comment by KenLee12323** [comment]
I think its in here 
https://github.com/KupynOrest/DeblurGAN/tree/master/checkpoints/experiment_name

**Comment by shreya2000** [comment]
> I think its in here
> https://github.com/KupynOrest/DeblurGAN/tree/master/checkpoints/experiment_name

No, I don't think it's available in this folder. Did you check it recently ?
